CHANGELOG.md

```diff
@@ -5,6 +5,89 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 
+## [unReleased] - 2021-MM-DD
+
+### Added
+
+-
+
+
+-
+
+
+-
+
+
+-
+
+
+-
```

Comment:
I have added a few placeholders per section so that we don't bottleneck upcoming PRs.

If we have one line, means that we will be getting conflicts until these sections are filled

Reply:
I feel it will be rather a source of collision with the `-` maybe rather have blank lines only?

Reply:
Could do blank lines too. Added the `-` so other PRs respect the two newline rule